tests/test_verifylib.py

```diff
@@ -1023,6 +1037,20 @@ def test_verify_passing_double_signed_layout(self):
             [self.alice_path, self.bob_path]
         )
         in_toto_verify(layout, layout_key_dict)
+    
+    def test_verify_failing_layout_current_datetime(self):
+        with patch('datetime' + '.date') as mock_date:
+            mock_date.now.return_value = datetime.datetime(2017, 6, 5, 4, 3, 2) 
+            mock_date.side_effect = lambda *args, **kw: date(*args, **kw)
+
+            assert datetime.date.now() == datetime.datetime(2017, 6, 5, 4, 3, 2) 
+            assert datetime.date(2009, 6, 8) == date(2009, 6, 8)
+            """Test pass verification with current date expiry layout."""
+            layout = Metablock.load(self.layout_current_datetime_expiry_path)
+            layout_key_dict = import_publickeys_from_file([self.alice_path])
+            with self.assertRaises(LayoutExpiredError):
```

Comment:
This currently passes because the expiry is set to `now()` at set up time and the check happens a few moments later when the test is executed. We want to test for the change in behavior. I recommend mocking the expiry check for two layouts:
1. The layout expiry and the test time are identical, resulting in an expected error (earlier, the two being the same would not cause an error)
2. The layout expiry is a second after the test time (mocked), resulting in no error.